coroutines-bluebird.js

```diff
@@ -29,8 +29,14 @@ const walk = bluebird.coroutine(function*(dir) {
 
 const main = () => {
   walk(process.argv[2])
-    .catch(s => process.stderr.write(s))
-    .then(s => process.stdout.write(s));
+  .then(data => {
+          process.stdout.write(data);
+          process.exit(0);
+        },
+        err => {
+          process.stderr.write(String(err) + '\n');
+          process.exit(1);
+        });
```

Comment:
In the following snippet if the then clause throws an error it will be silently rejected
```javascript
.then(data => {"something with data"}, err => {"something with err"});
```
In this version it will show up in the `catch`
```javascript
.then(data => {"something with data"})
.catch(err => {"something with err"})
```